File: runtime/Java/src/org/antlr/v4/runtime/atn/StarBlockStartState.java
/*
 * Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
 * Use of this file is governed by the BSD 3-clause license that
 * can be found in the LICENSE.txt file in the project root.
 */

package org.antlr.v4.runtime.atn;

/** The block that begins a closure loop. */
public final class StarBlockStartState extends BlockStartState {

	@Override
	public int getStateType() {
		return STAR_BLOCK_START;
	}
}

File: runtime/CSharp/runtime/CSharp/Antlr4.Runtime/Atn/StarBlockStartState.cs
/* Copyright (c) 2012-2017 The ANTLR Project. All rights reserved.
 * Use of this file is governed by the BSD 3-clause license that
 * can be found in the LICENSE.txt file in the project root.
 */
using Antlr4.Runtime.Atn;
using Antlr4.Runtime.Sharpen;

namespace Antlr4.Runtime.Atn
{
    /// <summary>The block that begins a closure loop.</summary>
    /// <remarks>The block that begins a closure loop.</remarks>
    public sealed class StarBlockStartState : BlockStartState
    {
        public override Antlr4.Runtime.Atn.StateType StateType
        {
            get
            {
                return Antlr4.Runtime.Atn.StateType.StarBlockStart;
            }
        }
    }
}
